admin-dev/themes/new-theme/scss/pages/product/product_page.scss

```diff
@@ -109,6 +109,112 @@ $product-page-padding-bottom: 80px !default;
   }
 
   // Components
+  .product-header-v2 {
+    display: flex;
+    flex-direction: row;
+    gap: 25px;
+    padding: 0.625rem;
+    margin-bottom: 1rem;
+    background: $white;
+    @include border-radius(0.375rem);
+    box-shadow: 0 0 12px rgba(0, 0, 0, 0.15);
+
+    > * {
+      flex-grow: 1;
+    }
+
+    $product-cover-size: 110px;
+
+    .product-header-cover {
+      max-width: $product-cover-size;
+
+      .image_preview-widget {
+        width: $product-cover-size;
+        height: $product-cover-size;
```

Comment:
Avoid using fixed height. If image is bigger for any reason, you will have trouble, just let it flow ;-)

Reply:
The widget container has a fixed height, but the image inside is floating It has auto-size but max-width and height The purpose is to let it adapt inside a restricted container